src/app/app.module.js

```diff
@@ -5,7 +5,9 @@ angular.module('esn.inbox-calendar', [
   'esn.notification',
   'esn.i18n',
   'esn.calendar.libs',
-  'esn.resource.libs'
+  'esn.resource.libs',
+  'linagora.esn.videoconference',
+  'linagora.esn.videoconference.calendar'
```

Comment:
Where are these modules loaded?

Reply:
They are loaded by using this `require('esn-frontend-videoconference-calendar/src/index.js');` in `src/index.js` below